templates/ai-ml.html

```diff
@@ -1,374 +1,417 @@
+        <p>NVIDIAとCanonicalは、あらゆるパブリッククラウド、オンプレミス、IoT機器でAIハードウェアアクセラレーションを提供できるよう協力しています。</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">機械学習向けに最適化されたNVIDIA DGXサーバーはUbuntuを搭載しており、Canonicalによるサポートが含まれています</li>
+          <li class="p-list__item is-ticked">
+            ハードウェアアクセラレーションのためのNVIDIA Kubernetesエクステンションは、CanonicalのKubernetesディストリビューションで有効化されています
+          </li>
+          <li class="p-list__item is-ticked">NVIDIA TegraとDrivePX2にはエッジAI向けUbuntuが搭載されています</li>
+        </ul>
+      </div>
     </div>
-  </div>
-</section>
-<section class="p-strip is-bordered" id="install-kubeflow">
-  <div class="row">
-    <div class="col-12">
-      <h2>Kubeflowを自分でインストールする</h2>
-      <p>Canonicalは開発者や運用者を支援するために継続的なベストプラクティスの革新と開発を行っています。</p>
-      <h3>KubeflowをKubernetesにデプロイ</h3>
-      <ol class="p-stepped-list--detailed">
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">Kubernetesのインストール</h3>
-          <div class="p-stepped-list__content">
-            <p>KubeflowはKubernetes上で動作します。Canonical KubernetesがKubeflowの最適な基盤です。</p>
-            <p>Kubernetesを独自にインストールするには、<a href="https://ubuntu.com/kubernetes/install" class="p-link--external">Kubernetesのインストール</a>のページにアクセスし、インストールガイドに従ってください。</p>
-          </div>
-        </li>
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">MicroK8sのインストール</h3>
-          <div class="p-stepped-list__content">
-            <p>MicroK8sをインストールするbashスクリプトを入手します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>wget https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/b783a95d-install-kubeflow-pre-micro.sh -O install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-            <p>スクリプトを実行できるようにパーミッションを設定します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>chmod a+x install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-            <p>スクリプトをルートとして実行します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>sudo ./install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-          </div>
-        </li>
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">環境変数の設定</h3>
-          <div class="p-stepped-list__content">
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>export KUBECONFIG=/snap/microk8s/current/client.config</code></pre>
-            </div>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>export GITHUB_TOKEN=${YOUR_GITHUB_TOKEN}</code></pre>
-            </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--25-75">
+      <hr />
+      <div class="col">
+        <div class="p-image-container u-hide--small">
```

Comment:
```suggestion
        <div class=" u-hide--small">
```
You don't need the image container classes here at all actually, `p-image-container` aligns to center which is not what we want here. It also looks like you need a new version of this logo as even with the container class removed it still has too much top and bottom whitespace to align correctly with the text (logos meant for logo sections can't be used in this component for this reason, you can tell the difference because those will always have this top and bottom spacing built in).
![image](https://github.com/user-attachments/assets/1b4f5469-b58e-478b-aab9-980a7e8db8e9)

It should align like so: 
![image](https://github.com/user-attachments/assets/622886f5-08d6-4e41-b229-9156460f087f)